ivy_tests/test_ivy/test_frontends/test_numpy/test_linear_algebra/test_matrix_and_vector_products.py

```diff
@@ -12,6 +12,63 @@
 )
 
 
+@st.composite
+def _get_dtype_value1_value2_axis_for_cross(
```

Comment:
Could you please move your helper strategy to be on top of your test? like how it's [here](https://github.com/unifyai/ivy/blob/23103f5adb8d7bc9a4da63ae5fb9a098dd695ce1/ivy_tests/test_ivy/test_frontends/test_numpy/test_linear_algebra/test_norms_and_other_numbers.py#L14-L94)

Reply:
Done! Thought all helpers should be before any test functions